CodeSniffer/Standards/Generic/Sniffs/Formatting/DisallowLongArraySyntaxSniff.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/**
+ * Generic_Sniffs_Formatting_DisallowLongArraySyntaxSniff.
+ *
+ * PHP version 5
+ *
+ * @category  PHP
+ * @package   PHP_CodeSniffer
+ * @author    Xaver Loppenstedt <xaver@loppenstedt.de>
+ * @copyright 2013-2015 Xaver Loppenstedt
```

Comment:
Unfortunately in didn't find a concrete copyright/copyleft guideline.
That's why I looked a several files like  OneClassPerFileSniff and ClosingPHPTagSniff and decided to put a copyright in similar form.